### PR TITLE
add manatee

### LINKED ISSUE #1415: New concept: MANATEE
There are three conceptlists that have manatee/manatí as an entry, but we do not have this mapped yet. We are also working with other datasets that will also feature this word. These are the forms in question:

- https://concepticon.clld.org/values/KochGrunberg-1914-821-439
- https://concepticon.clld.org/values/Gerardi-2022-705-304
- https://concepticon.clld.org/values/Bowern-2021-342b-188

If we agree that we should add this, I can prepare a PR.

### PR DESCRIPTION
**Pull request by FredericBlum**
# Pull request checklist

- [ ] add new concept list
- [ ] add new metadata
- [x] add new Concepticon concept sets
  * [x] checked whether the new concept(s) can be applied to existing lists with
    `concepticon notlinked --gloss "NEW_GLOSS"`
- [ ] add new Concepticon concept relations
- [ ] refine existing Concepticon concept set mappings
- [ ] refine Concepticon glosses
- [ ] refine Concepticon concept relations
- [ ] refine Concepticon concept definitions
- [ ] retire data

## Additional information

Closes #1415 . I have added this as AMAZONIAN MANATEE to make clear we refer exclusively to the South American species. All three conceptlists exclusively refer to South American language varieties (checked huntergatherer through SQLite, all references are from South America). @AnnikaTjuka Did I forget anything? It's been a long time since I last added concepts.

### CODE REVIEW

File: concepticondata/concepticon.tsv [hunk 1]
```diff
@@ -3978,3 +3978,4 @@ ID	GLOSS	SEMANTICFIELD	DEFINITION	ONTOLOGICAL_CATEGORY	REPLACEMENT_ID
 3992	SARDINE	Animals	A pilchard or other fish in the herring family.	Person/Thing	
 3993	JINGLE (INSTRUMENT)	Speech and language	One of a cluster of small bells, shaken or tapped on the palm of the player's hand. The small metal discs arranged around the frame of a tambourine are also called "jingles".	Person/Thing	
 3994	PIANO (INSTRUMENT)	Speech and language	A large musical instrument with a row of black and white keys that are pressed to play notes.	Person/Thing
+3995	AMAZONIAN MANATEE	Animals	Trichechus inunguis ()GBF ID: 2435296), one of the four living Manatee species. Native to the Amazon river system, the amazonian manatee is a fully aquatic mammal.	Person/Thing
```
Comment (AnnikaTjuka) [inline concepticondata/concepticon.tsv:3981]: There is one bracket too much and could you also add the link to the ID?

**Review by AnnikaTjuka — CHANGES_REQUESTED** [review]
I suggest that we add the more general species (MANATEE) since the list by Claire Bowern collects data on Australian languages so we wouldn't be able to link it to the Amazonian Manatee.

File: concepticondata/concepticon.tsv (hunk 1 above)
Comment (FredericBlum) [reply concepticondata/concepticon.tsv:3981]: Sure, I will change this

**Comment by FredericBlum** [comment]
> I suggest that we add the more general species (MANATEE) since the list by Claire Bowern collects data on Australian languages so we wouldn't be able to link it to the Amazonian Manatee.

Ok, even though the data for 'manatee' in the database only refers to South American languages? Then I will change this accordingly.

**Comment by FredericBlum** [comment]
I meant the huntergatherer database. All cases of 'manatee' in Bowern's data are from South American languages, which is why I proposed AMAZONIAN MANATEE.

**Comment by AnnikaTjuka** [comment]
I thought the database included languages spoken by hunter-gatherers in Australia, North America and South America. So is this list specific to South American languages?

**Comment by xrotwang** [comment]
The list isn't, but the languages which are coded for Manatee are exclusively the South American ones.

**Comment by AnnikaTjuka** [comment]
I see. Then we can leave it as is.

**Comment by FredericBlum** [comment]
I have corrected the GBF ID (Amazonian Manatee, not West Indian Manatee) and removed the extra bracket.

**Pull request merged**